mail_activity_board/views/mail_activity_view.xml

```diff
@@ -0,0 +1,216 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+<!--
+    VIEWS
+-->
+
+
+    <!-- FORM VIEW -->
+    <record id="mail_activity_view_form_board" model="ir.ui.view">
+        <field name="name">mail.activity.form.boards.sdi</field>
+        <field name="model">mail.activity</field>
+        <field name="priority">30</field>
+        <field name="arch" type="xml">
+            <form string="Activity Form" create="false" edit="false" delete="false">
+                <sheet string="Activity">
+                    <group invisible="1">
+                        <field name="activity_category" invisible="1" />
+                        <field name="res_model" invisible="1"/>
+                        <field name="res_model_id" invisible="1"/>
+                        <field name="res_id" invisible="1"/>
+                        <field name="previous_activity_type_id"/>
+                        <field name="has_recommended_activities"/>
+                    </group>
+                    <group attrs="{'invisible': [('has_recommended_activities','=',False)]}">
+                        <div>
+                            <p><strong>Recommended Activities</strong></p>
+                            <field name="recommended_activity_type_id" widget="radio"
+                                domain="[('previous_type_ids', '=', previous_activity_type_id)]"
+                                options="{'horizontal':true}"
+                                nolabel="1"/>
+                        </div>
+                    </group>
+                    <group>
+                        <group>
+                            <field name="activity_type_id" required="1" options="{'no_create': True, 'no_open': True}"/>
+                            <field name="res_model_id_name"/>
+
+                            <button name="open_origin" type="object" class="centre oe_link" nolabel="1">
+                                <field name="res_name"><field name="res_model_id_name"/></field>
+                            </button>
+
+                            <field name="calendar_event_id" invisible="1"/>
+                            <field name="create_date" invisible="1"/>
+                            <field name="summary" placeholder="e.g. Discuss proposal"/>
+                        </group>
+                        <group>
+                            <field name="date_deadline"
+                                   attrs="{'invisible': [('activity_category', '=', 'meeting')]}"/>
+                            <field name="calendar_event_id_start" string="Start meeting"
+                                   attrs="{'invisible': [('calendar_event_id','=', False)], 'readonly': 1}"/>
```

Comment:
`1` is not a domain. Instead, add directly the XML attribute `readonly="1"`

Reply:
I think that the "readonly" attribute is not necessary because I have already defined the form as non-editable, so I have deleted it from the fields.